### PR TITLE
v1.36 Mobile Changelog

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/administration/mobile-changelog.md [hunk 1]
```diff
@@ -1,5 +1,44 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.36.0 Release
+- Release Date: October 16, 2020
+- Server Versions Supported: Server v5.25+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - **Upgrade to server version v5.25 or later is required.** Support for server [Extended Support Release](https://docs.mattermost.com/administration/extended-support-release.html) (ESR) 5.19 has ended and upgrading to server ESR v5.25 or later is required. As we innovate and offer newer versions of our mobile apps, we maintain backwards compatibility only with supported server versions. Users who upgrade to the newest mobile apps while being connected to an unsupported server version can be exposed to compatibility issues, which can cause crashes or severe bugs that break core functionality of the app.
```
Comment (amyblais) [inline source/administration/mobile-changelog.md:8]: Update in app stores.

File: source/administration/mobile-changelog.md [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Improvements
+ - Added **Channel Settings > Notification Preferences** to add the ability to edit mobile push notification settings at the channel level.
+ - Server URL now autofills when opening the app from the mobile landing page.
```
Comment (enahum) [inline source/administration/mobile-changelog.md:14]: I can assume this is the landing page of the webapp, but it's not very clear imho

File: source/administration/mobile-changelog.md [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+ - Improved styling of Read Only channels.
+ - Removed the filename from an error message when an image/video was too large.
+ - Improved unread badge styling of the hamburger menu and team icons.
+ - Improved styling of autocomplete modals.
+ - Improved the validation error message of the Enter Server URL screen when entering an invalid server URL.
+
+### Bug Fixes
+
+#### All apps
+ - Fixed an issue where a hashtag (#) character added to an announcement banner caused the app to display a blanck screen.
```
Comment (enahum) [inline source/administration/mobile-changelog.md:28]:
```suggestion
 - Fixed an issue where a hashtag (#) character added to an announcement banner caused the app to display a blank screen.
```

File: source/administration/mobile-changelog.md [hunk 4]
```diff
@@ -1,5 +1,39 @@
...
+ - Fixed an issue where users were still able to upload files via the share extension when ``EnableMobileFileUpload`` was disabled on the server.
+ - Fixed an issue where a draft message on the reply thread was not retained if the user navigated away from the thread.
+ - Fixed an issue where a thumbnail of a file attachment posted in a reply thread displayed in the center channel.
+ - Fixed an issue where users were unable to join public channels via channel permalinks.
```
Comment (enahum) [inline source/administration/mobile-changelog.md:32]:
```suggestion
 - Fixed an issue where users were unable to join public channels via channel link.
```

**Review by enahum — CHANGES_REQUESTED** [review]

File: source/administration/mobile-changelog.md (hunk 1 above)
Comment (amyblais) [reply source/administration/mobile-changelog.md:8]: Sounds good to me.

**Review by enahum — APPROVED** [review]

**Review by esethna — APPROVED** [review]

**Pull request merged**